tests/outputs/a_manurev/refereeReports.yaml

```diff
@@ -1,21 +1,27 @@
 1902cl_actaArev:
   claimed_found_what:
+    -
   claimed_why_important:
+    -
   did_how:
+    -
   did_what:
-  due_date: '2020-09-15'
+    -
+  due_date: ''
   editor_eyes_only: ''
   final_assessment:
-  first_author_last_name: Einstein
+    -
+  first_author_last_name: ''
   freewrite: ''
   journal: ''
-  month: 06
+  month: ''
   recommendation: reject
-  reviewer: zcliu
+  reviewer: ''
```

Comment:
this should be passed in and maybe loaded from the user config.  we can discuss this.